openshift/system/Dockerfile.on_prem

```
@@ -62,7 +64,8 @@ RUN export ${BUNDLER_ENV} >/dev/null \
     && chmod g+w -vfR log tmp public/assets db/sphinx \
     && umask 0002 \
     && cd /opt/system \
-    && npm install --no-progress \
+    && npm install -g yarn \
+    && yarn install --no-progress \
```

This doesn't seem to do anything 🤔 neither does `--silent` (I tried locally)
```suggestion
    && yarn install \
```

---

Well, it doesn't appear in the logs of circle, OS and capistrano... so seems to be working